lib/portage/sync/modules/git/git.py

```diff
@@ -285,11 +290,13 @@ def verify_head(self, revision="-1"):
 
             rev_cmd = [self.bin_command, "log", "-n1", "--pretty=format:%G?", revision]
             try:
+                kwargs = self.spawn_kwargs.copy()
+                kwargs["env"] = env
                 status = portage._unicode_decode(
-                    subprocess.check_output(
+                    portage.process.check_output(
```

Comment:
This call is problematic. The files gnupg files generated by gemato will be owned by root, but we need to run git as non-root to avoid the "'/var/db/repos/gentoo' is owned by someone else" error.

I think we would need to invoke the gemato setup code (`self._get_openpgp_env()`) as the appropriate non-root user.